test/rest_request_template/travis_correct_snowpipe_streaming_string_json.json

```diff
@@ -14,6 +14,7 @@
     "snowflake.schema.name": "SNOWFLAKE_SCHEMA",
     "snowflake.role.name": "SNOWFLAKE_ROLE",
     "snowflake.ingestion.method": "SNOWPIPE_STREAMING",
+    "snowflake.streaming.enable.single.buffer": "$SNOWFLAKE_STREAMING_ENABLE_SINGLE_BUFFER",
```

Comment:
This line is going to be added for each `SNOWPIPE_STREAMING` test.

Reply:
Why introduce a new way of introducing parameters with '$'?

Reply:
Using caps only doesn't make the template parameters stand out. 
In this example `SNOWFLAKE_SCHEMA` and `SNOWFLAKE_ROLE` are parameter names while `SNOWPIPE_STREAMING` is a value.

I'm also inclined to adjust the naming for all template parameters.